### PR TITLE
registerPage email텍스트필드와 유효성 검사 작동하는 것 작업입니다.

### PR DESCRIPTION
**Pull request by chepchep2**
RegisterPage 
- 이메일을 입력하는 텍스트필드 작성 
- 이메일을 입력할때 유효성 검사
  - 시작, 작성시 에러메세지 안뜨고 이메일 형식이 아닌데 next나 텍스트필드를 벗어나면 에러메세지가 뜹니다.

### CODE REVIEW

**Comment by chepchep2** [comment]
* **#33** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/33?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#32** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/32?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#31** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/31?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#30** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/30?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#29** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/29?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#28** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/28?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#27** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/27?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#26** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/26?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#25** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/25?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#24** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/24?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#23** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/23?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#22** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/22?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#21** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/21?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#20** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/20?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#19** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/19?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#18** <a href="https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/18?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `feature/introPage`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @chepchep2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: lib/common/auth_text_input_widget.dart [hunk 1]
```diff
@@ -23,47 +25,69 @@ class AuthTextInputWidget extends StatelessWidget {
...
-        crossAxisAlignment: CrossAxisAlignment.start,
+      child: Stack(
         children: [
-          Text(
-            labelText,
-            style: const TextStyle(
-                color: AppColor.primaryBlue1,
-                fontSize: 12,
-                fontWeight: FontWeight.w400),
+          Positioned(
```
Comment (flowkater) [inline lib/common/auth_text_input_widget.dart:42]: Positioned 로 혹시 지정한 이유가 따로 있나요?

File: lib/page/register_page.dart [hunk 2]
```diff
@@ -37,10 +31,16 @@ class _RegisterPageState extends State<RegisterPage> {
   }
 
   void updateStatus() {
-    setState(() {
-      isEmailValid = validateEmail(emailController.text);
-      isPasswordValid = validatePassword(passwordController.text);
-    });
+    final email = emailController.text;
```
Comment (flowkater) [inline lib/page/register_page.dart:34]: onEditingComplete를 콜백으로 validate는 한번만 호출되면 좋을 것 같은데
이렇게 하면 매번 동작을 하는 것 같아요.

**Review by flowkater — CHANGES_REQUESTED** [review]
onEditingComplete를 사용하는
방법을 다시 고민해보면 좋을 것 같습니다!

File: lib/common/auth_text_input_widget.dart (hunk 1 above)
Comment (chepchep2) [reply lib/common/auth_text_input_widget.dart:42]: 위치를 제대로 못잡아서 Positiond를 썼습니다..

**Comment by chepchep2** [comment]
### Merge activity

* **Mar 17, 8:12 PM EDT**: @chepchep2 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/27).
* **Mar 17, 8:41 PM EDT**: [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/27) rebased this pull request as part of a merge.
* **Mar 17, 8:43 PM EDT**: @chepchep2 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/team-stoodi/MyBaseballRecordClient/27).

**Pull request merged**